web/src/main/java/org/mskcc/cbio/portal/web/ProxyController.java

```diff
@@ -54,7 +54,7 @@
   public void setBitlyURL(String property) { this.bitlyURL = property; }
 
   private String sessionServiceURL;
-  @Value("${session.service.url}")
+  @Value("${session.service.url:}")
```

Comment:
This might be easier to understand than a single colon: http://stackoverflow.com/a/33687265/2464448 . I think GlobalProperties class is an anti-pattern, so this looks good to me, but maybe we can change it to @Value("${session.service.url:#{null}}") like in the answer that I gave the link to.